R/plotly_data.R

```diff
@@ -104,116 +104,161 @@ highlight_key <- function(...) {
   crosstalk::SharedData$new(...)
 }
 
+# ---------------------------------------------------------------------------
+# dplyr methods
+# ---------------------------------------------------------------------------
+
 #' @rdname plotly_data
-#' @export
 groups.plotly <- function(x) {
-  dplyr::groups(plotly_data(x))
+  groups(plotly_data(x))
 }
 
 #' @rdname plotly_data
-#' @export
 ungroup.plotly <- function(x, ...) {
```

Comment:
This is now unexported?  I'm fine with it as long as it still works

Reply:
Yea, the method is now registered in `.onLoad()` instead